### ISSUE TITLE
Medallia feedback button and modal a11y issues

### ISSUE THREAD

**Issue by aubreyarcangel** [department-of-veterans-affairs/va.gov-team#52134]
capturing some issues from a recent [a11y audit of Medallia feedback button and modal](https://docs.google.com/document/d/1Wo-rQqBtCCWbgHROtyb5lLbZtkD1E9myq6xKVZFT1N4/)

From @joshkimux: 
- 2 of our a11y specialists conducted a cursory audit of the medallia feedback intercept survey and feedback button. They caught a couple more issues that wouldn’t typically emerge from a standard 508 audit as they’re based around WCAG 2.1 AA+ criteria (basically, accounting for mobile/responsive which is the recent VA.gov standard that came into play after this was originally audited a while back).
- Issues around magnification and responsiveness were also caught in appended tasks in a recent assistive tech study. Of the 2 participants who used zoom tools, both were unable to notice the feedback button due to its positioning on the bottom right hand of the screen. Often times these folk can only see half of the screen at any given time and read left to right-- meaning that the feedback button is almost impossible to identify.

**Comment by joshkimux** [comment]
Added some labels here ❤️ These issues would constitute a `defect-level-2`; which are relatively major. I'd recommend looking to address these sooner than later 🙏 If y'all need support, please reach out to the CAIA team.

**Comment by sara-amanda** [comment]
## CAIA Closing Ticket 
- [x] Closing per @aubreyarcangel 
> The unable to exit modal issue has been fixed.
### Remaining a11y recommendations

- [ ] #58574
- [ ] #58577

cc: @eli-oat
